app/src/main/java/ch/epfl/sdp/contamination/ConcreteDataReceiver.java

```diff
@@ -175,4 +179,8 @@ public void getMyLastLocation(Account account, Callback<Location> callback) {
             }
         });
     }
+
+    public void getNumberOfSickNeighbors(String userId, Callback callback){
```

Comment:
Should not use the generic type of Callback without type parameter

Reply:
That's why you are always using typer parameters ! Ok I'll fix it.